packages/components/src/tooltip/index.stories.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { withKnobs } from '@storybook/addon-knobs';
+import Tooltip from './index';
+import styles from './index.stories.module.scss';
+
+const tooltipShort = 'Design is like a joke.';
+
+export default { title: 'Tooltip', component: Tooltip, decorators: [withKnobs] };
```

Comment:
please try if it still works if you change `text` or `children` after initial render

Reply:
and try running it with ssr storybook

Reply:
still valid